src/views/Shopping.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { useReducer } from 'react'
+import AddItem from '../components/AddItem'
+import ListItems from '../components/ListItems'
+import './Shopping.css'
+
+export default function Shopping() {
+    const initialItemList = [
```

Comment:
with the useReducer hook lines 8-36 traditionally are before the function is declared, so like starting on line 6